lib/common/prefix.go

```diff
@@ -27,4 +27,5 @@ const (
 	BlockAccountSequenceIDByAddressPrefix = string(0x33)
 	BlockAccountPrefixFrozen              = string(0x34)
 	TransactionPoolPrefix                 = string(0x40)
+	TmpPrefix                             = string(0x50) // temporary used data
```

Comment:
Is there a better name instead of Tmp?

Reply:
@kfangw it is renamed to `InternalPrefix` :)